styles/index.css

```diff
@@ -0,0 +1,108 @@
+body {
+    background-color: rgb(255, 255, 255);
+}
+
+.tempbutton {
+    background-color: transparent;
+    border: none;
+    font-size: 45px;
+}
+
+.city_button {
+    background-color: transparent;
+    border: none;
+    font-size: 25px;
+}
+
+.header {
+    text-align: center;
+    font-family: "American Typewriter";
+    font-size: 25px;
+}
+
+.city-header {
+    font-family: "Gill Sans";
+    font-size: 45px;
+}
+
+.grid-container {
+    display: grid;
```

Comment:
Amazing work with styling 👍🏾 I would suggest adding media queries to account for all types of screen sizes